session/objecthandling.go

```diff
@@ -50,7 +50,12 @@ func init() {
 	if err := GlobalObjectHandler.RegisterHandler("container", &ContainerHandler{}, false); err != nil {
 		panic(err)
 	}
-
+	if err := GlobalObjectHandler.RegisterHandler("sn-layout-container", &LayoutContainerHandler{}, false); err != nil {
+		panic(err)
+	}
+	if err := GlobalObjectHandler.RegisterHandler("masterobject", &MasterObjectHandler{}, false); err != nil {
+		panic(err)
+	}
```

Comment:
Old pattern, but just to double check: Is this safe as long as it  does not panic in the unit tests?

Reply:
not sure what you mean with "safe" it should only ever panic during start up of the program, not otherwise during runtime. And yes that would include unit tests for that package.

Reply:
i.e. it's very "fail fast"

Reply:
Cool! 😎